llvm/test/Transforms/LoopVectorize/AArch64/scalable-struct-return.ll

```diff
@@ -0,0 +1,97 @@
+; RUN: opt < %s -mattr=+sve -passes=loop-vectorize,dce,instcombine -force-vector-interleave=1 -prefer-predicate-over-epilogue=predicate-dont-vectorize -S | FileCheck %s
```

Comment:
Do we really need the `dce,instcombine` passes here? I understand it cleans up the IR, but it can also make it harder to see what's going on. There are two examples where it can be a problem:

1. Somebody changes the output of the loop vectoriser, but it doesn't show up here because the end result after running instcombine is the same. That may not itself be a problem, but it does make it harder to see what effect code changes had on the output.
2. Somebody adds a patch that changes DCE or InstCombine passes and is then surprised to see a loop vectoriser test failing. Effectively this RUN line is testing 3 different passes at once and increases the likelihood of churn in the file.

Reply:
It's not needed in this PR (since we're not checking the IR anyway yet), so I've removed it :+1: I think previously, this made  the CHECKs a little easier, but I don't remember (been a long time since I first wrote these tests :sweat_smile:).